### PR TITLE
Add NEWS file

### LINKED ISSUE #257: Packaging
There is no change log or NEWS file in the release.  How do users who build from source know what bugs have been fixed or new features added?  Note that a list of every commit is not very convenient.  A summary is much more useful for many users.

When downloading, the package name that is saved is libblockdev-libblockdev-2.10-1.tar.gz
The first  libblockdev- is redundant.  Indeed, then the tarball is expanded the package does expand to libblockdev-libblockdev-2.10-1/ but this type of packaging is quite unusual and should be changed to conform to the name listed on the releases page.

### PR DESCRIPTION
**Pull request by vojtechtrefny**
Add NEWS.rst file with release information.

Fixes #257

### CODE REVIEW

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

File: Makefile.am [hunk 1]
```diff
@@ -100,9 +100,6 @@ coverage-all: all
 		$(COVERAGE) report --include="src/*" > coverage-report.log
```
Comment (vpodzime) [inline Makefile.am]: *onw*?

Comment (vojtechtrefny) [reply Makefile.am]: Fixed, thanks.

**Pull request merged**